### DISCUSSION TITLE
Feature Request: System Tray, Manual Input of Region Resolution and Region Location

### DISCUSSION THREAD

**Discussion by forrmatt** [tom-englert/RegionToShare#13] [Ideas]
First, great tool, thank you. I had been looking for something similar to this since I upgraded to an Ultrawide. This feature request combines a few elements of the other requests that have come in up to this point, but I will try to sum them up together as they are dependent on the previous.

1. System Tray Icon 
As mentioned in another request, having a System Tray icon would be a great bonus for this tool. It would allow for some additional input menus as well as an easy place to close the app.
The immediate menu items I see as beneficial for the System Tray menu would be: Exit, Region Resolution, Region Location, (Bonus Item) Profile
![image](https://user-images.githubusercontent.com/22401463/151848207-05583456-6197-4767-afda-7ee6e1c5d713.png)

2. Manual Input of Region Resolution
A small menu in the system tray menu that allows the user to manually input the desired region by pixel count in by X and Y sizes (2560x1440, etc...). This would ensure that I can share the exact dimensions required, with or without the use of FancyZones or another Virtual Monitor setup. 
![image](https://user-images.githubusercontent.com/22401463/151848156-d1fc4ff2-23fc-42b3-a243-e7f8b1285bfa.png)

3. Manual Input of Region Location
Much like above, a small system tray sub-menu to input the location of where the Region is location. It would be ideal if I could give specific coordinate locations of where the Region should start. Assume top left of region is start. The menu would provide the ability to select the monitor in which is being input and an X,Y pixel location of where the top-left of the Region is located. A default of 0,0 would ensure that it starts at the top-left of the selected monitor.
![image](https://user-images.githubusercontent.com/22401463/151848319-b3ade9c5-24bd-4f72-b551-a966794a3e04.png)

4. BONUS ITEM! If RegionToShare could enable some basic profile functionality, that would ensure users could easily switch between different predetermined Regions (both locations and sizes) to fit the needs of what is to be presented. The system tray could allow for saving and loading of preset profiles, as well as creating new Profiles. The profiles XML only need to save a name, region resolution, and region starting location. XML can be easily written to local app data to prevent any permissions issues. Then they can be easily accessed by the user via the System tray to bounce between various configurations.
![image](https://user-images.githubusercontent.com/22401463/151848815-8fa6b580-ae50-4fe6-b353-0cb616439a7a.png)
![image](https://user-images.githubusercontent.com/22401463/151848872-43fc8f45-6dd4-4137-930d-a4d769bdc870.png)
![image](https://user-images.githubusercontent.com/22401463/151849346-4410cb38-5532-4198-9290-6a38c75bc0bd.png)

**Comment by tom-englert** [comment]
Good points, do you like to volunteer to implement this?
